### PR TITLE
Adding 8.2.1 support

### PR DESCRIPTION
**Pull request by k-bx**

### CODE REVIEW

**Comment by k-bx** [comment]
It passed! @tmcgilchrist please review

File: .stack-8.0.2.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+resolver: lts-9.1
+packages:
+- '.'
+- protobuf/
+- protobuf-lens/
```
Comment (tmcgilchrist) [inline .stack-8.0.2.yaml]: Could you either merge this file with `stack.yaml` making the default GHC 8.0.2 (you'd need to change the `.travis.yaml` file back) that way a fresh checkout would just work with `stack build`.

**Comment by tmcgilchrist** [comment]
Might be good to add a line in the change log pointing out the gcc 8.2 support.

Thanks for adding this. 👍

**Comment by k-bx** [comment]
@tmcgilchrist all right, done!

**Comment by tmcgilchrist** [comment]
Thank you.

**Pull request merged**